tests/test_project/apps.py

```diff
@@ -86,3 +88,60 @@ def register_dashboard_charts(self):
                 'js': ('dashboard-test.js',),
             },
         )
+
+    def register_menu_groups(self):
+        auth_list = {
+            2: {
+                'label': 'Add user',
+                'model': 'auth.User',
+                'name': 'changelist',
+                'icon': 'add-icon',
+            },
+            1: {
+                'model': 'auth.User',
+                'name': 'add',
+                'icon': 'edit',
+                'label': 'Edit User',
+            },
+        }
+
+        auth_group = {
+            'label': 'Authentication And Authorization',
+            'items': auth_list,
+            'icon': 'user-icon',
```

Comment:
When I was using it, I realised, instead of setting icons like this, shouldn't we use https://icons.getbootstrap.com/ ? Users will have a lot more choices!

Reply:
@atb00ker Its good idea.
I have checked their [accessibility](https://icons.getbootstrap.com/#accessibility) page.
I think we can use this
```
<i class="bi-alarm" style="font-size: 2rem; color: cornflowerblue;"></i>
```
We will ask user to provide only class name. @nemesisdesign  and @atb00ker What do you think?

Reply:
Sounds good to me! :smile:

Reply:
Hey, please create a new issue for it and handle it seperately, otherwise we will be stuck in perpetual review loop.

Reply:
is this resolved?

Reply:
Created an issue https://github.com/openwisp/openwisp-utils/issues/206 for it.